### ISSUE TITLE
Error 405 - Method Not Allowed

### ISSUE THREAD

**Issue by HyperrJedi** [McCloudS/subgen#119]
I have subgen  and Jellyfin on the same docker network and use the docker ips to connect them. When I play media on Jellyfin that is in Turkish, which I want to be  translated into English, this shows up in subgen logs.

`subgen  | INFO:     172.18.0.2:36726 - "POST / HTTP/1.1" 405 Method Not Allowed`

In docker, I use the environment variables in the compose yaml. The pictures are my yaml setup and the logs. I'm sure this is a user error, I just  don't know where. Any help? Thanks!

![Subgen yaml](https://github.com/user-attachments/assets/555f6cf3-db80-4731-8787-46920134acce)
![subgen logs](https://github.com/user-attachments/assets/9fdcb85b-d090-48e5-8eff-5470b4584582)

**Comment by McCloudS** [comment]
At a glance, most of that seems fine.  How is your jellyfin webhook set up?

**Comment by McCloudS** [comment]
I just double checked it still works on the newest Jellyfin and Jellyfin webhook plugin.  I've attached a screenshot of my Jellyfin settings.  
![image](https://github.com/user-attachments/assets/c10f8af7-2a4d-4b22-a0e8-8764e7207942)
![image](https://github.com/user-attachments/assets/3c321783-8cce-4f8b-b9aa-2596421cdbe7)
![image](https://github.com/user-attachments/assets/c87317a4-5a97-4b1e-979b-6765177a43f7)
![image](https://github.com/user-attachments/assets/d0d6c718-7901-47ad-8084-40a2653238bf)

**Comment by HyperrJedi** [comment]
I forgot the /jellyfin in the webhook url lmao. Ok, that works. However, now when I play an episode of the Turkish show that has no subtitles, which I want English subtitles, I see nothing about it being translated in the logs. Looking at your logs, I don't get anything after the green INFO. Why could that be?

**Comment by McCloudS** [comment]
You need to set either of the procs to True depending on what you want.  
![image](https://github.com/user-attachments/assets/ce704cd5-91cc-48dd-9326-2ed386c84055)

**Comment by HyperrJedi** [comment]
Ok, I want it to translate and make the translated srt if I play media that doesn't already have either an external srt or embedded subtitles. The external srts have a .en for English. And I only want it when media is played, not when it's added. How would I change the environment variables?

**Comment by HyperrJedi** [comment]
Quick update. I added some stuff to the environ variables. I was able to get the episode translated in 27 minutes, which is pretty good for the episode wahich was 2 hours long. After that, i tried playing a show not in turkish with embedded subtitles. Subgen didn't mmake subtitles for it. I played a Turkish show with external srts. Subgen reported making and srt for it. The subtitle is named according with the episode (ex. S02E11.en.srt). How do I get Subgen to not do that?
![updated yaml](https://github.com/user-attachments/assets/d8580b3f-199c-4958-811c-9b5d8b18796b)
